app/components/NoSuggestionsPanel/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Icon from '../Icon'
+import { IntlMixin } from 'react-intl'
+
+/**
+ * Generic panel showing an icon and message, to
+ * use when there are no suggestions to display.
+ */
+
+let NoSuggestionsPanel = React.createClass({
+  mixins: [IntlMixin],
+
```

Comment:
I would like to use v2 or `react-intl`, which is now in beta-v2. For one thing, it removes the need for mixins.